stable/jenkins/values.yaml

```diff
@@ -5,8 +5,8 @@
 
 Master:
   Name: jenkins-master
-  Image: "gcr.io/kubernetes-charts-ci/jenkins-master-k8s"
-  ImageTag: "v0.7.0"
+  Image: "jenkins"
```

Comment:
Any reason not to use jenkinsci/Jenkins as what was outlined in #875 since it's updated as part of the jenimks reease procrss and no humans are needed to update the mirror of what docker considers official?